mx.truffleruby/mx_truffleruby.py

```diff
@@ -295,6 +295,13 @@ def ruby_testdownstream_sulong(args):
     jt('test', 'mri', 'zlib')
     jt('test', 'bundle')
 
+def mx_post_parse_cmd_line(opts):
+    if mx.suite("tools", fatalIfMissing=False):
```

Comment:
This needs to check if the suite is a source suite, as [Gilles said](https://github.com/oracle/truffleruby/pull/1258#discussion_r178041820):
```python
if mx.suite("tools", fatalIfMissing=False) and not _suite.isBinarySuite():
```